### DISCUSSION TITLE
Error Azure.Resource.UseTags Rule

### DISCUSSION THREAD

**Discussion by fabianldsk84** [Azure/PSRule.Rules.Azure#1977] [Q&A] [answered]
Hi guys..
I am getting this error in all the modules of my code:
"failed Azure.Resource.UseTags. Azure resources should be tagged using a standard convention"

It is clear what the error says but my bicep code uses the tags like this:

main .bicep file

param location string = resourceGroup().location
param tags object = resourceGroup().tags
param storageName string

module storageAccountModule 'modules/storage-account.bicep' = {
  name: 'storageAccountDeploy'
  params: {
    location: location
    tags:  tags
    name: storageName
  }
}

If I use the resourceGroup() function for location works fine  but for tags it doesnt work

Thanks

**Comment by BernieWhite** [comment, marked as answer]
@fabianldsk84 By default, no tags are set for a resource group (same as Azure). When testing configurations that rely on the resource group tags consider setting the [AZURE_RESOURCE_GROUP](https://azure.github.io/PSRule.Rules.Azure/setup/configuring-expansion/#deployment-resource-group) configuration.

For example:

```yaml
configuration:
  AZURE_RESOURCE_GROUP:
    tags:
      env: test
      app: test
```

Any reference to `resourceGroups().tags` will then use these tags during unit testing.

---
I hope that helps.

**Reply by fabianldsk84** [reply]
Thanks for your response Bernie
